Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule_test.py

```diff
@@ -137,3 +137,16 @@ def test_datestring_to_millisecond_timestamp():
     assert 1581341954000 == datestring_to_millisecond_timestamp(datesting5)
     assert 1581341954123 == datestring_to_millisecond_timestamp(datesting3)
     assert 1581341954123 == datestring_to_millisecond_timestamp(datesting4)
+
+
+def test_get_feed_config():
+    custom_fields_mapping = {
+        "old_field1": "new_field1",
+        "old_field2": "new_field2"
+    }
+    client = Client(
+        url="https://www.spamhaus.org/drop/asndrop.txt",
+        feed_url_to_config="some_stuff",
+        custom_fields_mapping=custom_fields_mapping
+    )
+    assert {} == client.get_feed_config()
```

Comment:
What are you testing here exactly?

Reply:
That when running the function with no input, something is coming back.

Reply:
Consider adding a comment there